ex8.1_hierarchy.py

```diff
@@ -0,0 +1,87 @@
+class RegularUser:
+    """
+    managing the regular users
+    """
+    def __init__(self, username, password, manager=False):
```

Comment:
This __init__ is redundant, read about dataclass decorator and use it - 
https://docs.python.org/3/library/dataclasses.html